doc/shell_heavy_guide.md

```diff
@@ -0,0 +1,244 @@
+# 1. Test Objective
+Shel_heavy test is a part of the shell test. The test cases that require big disk size or big memory or with heavy workload are categorized as shell_heavy.
+# 2. Deploy Regression Test Environment
```

Comment:
Hope that CUBRID developers can get guide to execute shell_heavy locally instead of regression deployment.

Reply:
I've added Execute shell_heavy Test.